### PR TITLE
Add tutorial 

### LINKED ISSUE #33: Needs an about/instructions page

### PR DESCRIPTION
**Pull request by schreiaj**
joyride

closes #33

### CODE REVIEW

File: src/containers/Editor.js [hunk 1]
```diff
@@ -13,8 +14,10 @@ import { extractStates } from '../transforms/Module';
 
 import { findAvailableKey, createSafeKeyFromName } from '../utils/keys';
 import { StateTemplates, ModuleTemplates } from '../templates/Templates';
+import { BasicTutorial, EditTutorial } from '../templates/Tutorial';
 
 import './Editor.css';
+import './Joyride.scss';
```
Comment (arscan) [inline src/containers/Editor.js]: If we are just using scss as a way to get to Joyride's styles in node_modules, we could do this instead:

```javascript
import '../../node_modules/react-joyride/lib/react-joyride-compiled.css'
```

Bringing in scss and only using it on one line of code seems a little wasteful, unless you think we should start doing stuff in scss.

**Comment by kjmahalingam** [comment]
I think this is a good, brief tutorial. It will require some user testing to see if there are further things to clarify, but this seems nice to me for now.

Pending a decision on whether to address @arscan's concerns above, I think this is a good addition.

**Comment by arscan** [comment]
Oh, I just realized that the poorly formatted css in `Editor.css` is from the css file I referenced in one of my comments.  Can we just include it as I pointed out or is there a reason why we have it copied locally (like, did you make minor changes or something?)

File: src/containers/Editor.js [hunk 2]
```diff
@@ -79,10 +93,38 @@ class Editor extends Component {
...
+        this.props.push('#' + key)
+
+      } catch (ex){
+        alert('Invalid module ' + ex.message);
+        return
+      }
+    }
+  }
+
+  joyrideCallback = (data) => {
```
[the remark this answers is not in the document]
Comment (schreiaj) [reply src/containers/Editor.js]: It's in there if we needed it in the future. I could pull it out though, it's not used anywhere.

**Comment by schreiaj** [comment]
@arscan I tried referencing it, was having issues. Just caved and did it the brute force way and never removed it. I'll give it another shot to see if I was just doing something wrong.

**Pull request merged**